lib/chef/chef_fs/file_system/repository/chef_repository_file_system_root_dir.rb

```diff
@@ -44,6 +44,7 @@
 require "chef/chef_fs/data_handler/user_data_handler"
 require "chef/chef_fs/data_handler/group_data_handler"
 require "chef/chef_fs/data_handler/container_data_handler"
+require "chef"
```

Comment:
I'd prefer simpler ruby code as well but doing file permissions for windows, without using file/dir providers, will likely be more complicated that what's here. From my chats with Windows people doing file permissions on Windows from Ruby is a pain.

Would you be opposed to using using `Dir.mkdir`/`File.open` as before but then leveraging https://github.com/chef/chef/blob/master/lib/chef/win32/security/securable_object.rb directly to handle the Windows perms?